plugins/language_haxe.lua

```diff
@@ -0,0 +1,91 @@
+-- mod-version:3
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "Haxe Compiler Arguments",
+  files = "%.hxml$",
+  patterns = {
+    { pattern = "[%-][%-%w_]*",      type="keyword"},
```

Comment:
I don't think the square brackets are needed in the first part.

Reply:
Fixed